pkg/edgedictionary/update.go

```diff
@@ -0,0 +1,60 @@
+package edgedictionary
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/compute/manifest"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/v2/fastly"
+)
+
+// UpdateCommand calls the Fastly API to describe a service.
+type UpdateCommand struct {
+	common.Base
+	manifest manifest.Data
+	input    fastly.UpdateDictionaryInput
+
+	newname common.OptionalString
+}
+
+// NewUpdateCommand returns a usable command registered under the parent.
+func NewUpdateCommand(parent common.Registerer, globals *config.Data) *UpdateCommand {
+	var c UpdateCommand
+	c.Globals = globals
+	c.manifest.File.Read(manifest.Filename)
+	c.CmdClause = parent.Command("update", "Update name of dictionary on a Fastly service version").Alias("get")
+	c.CmdClause.Flag("service-id", "Service ID").Short('s').StringVar(&c.manifest.Flag.ServiceID)
+	c.CmdClause.Flag("version", "Number of service version").Required().IntVar(&c.input.ServiceVersion)
+	c.CmdClause.Flag("name", "Old name of Dictionary").Short('n').Required().StringVar(&c.input.Name)
+	c.CmdClause.Flag("new-name", "New name of Dictionary").Required().Action(c.newname.Set).StringVar(&c.newname.Value)
```

Comment:
As explained in a separate conversation thread, `--new-name` is optional along with another flag not yet defined (but should be added here) `--write-only` and because there are two optional flags it means `--new-name` no longer is 'required' for this command to be useful.

```suggestion
	c.CmdClause.Flag("new-name", "New name of Dictionary").Action(c.newname.Set).StringVar(&c.newname.Value)
```

Reply:
Updated.